packages/core/cli.ts

```diff
@@ -23,7 +23,7 @@ import {
 	startFuzzingAsync,
 	stopFuzzingAsync,
 	printError,
-} from "./core";
```

Comment:
`core` is no ESM, so the suffix is not needed.

Reply:
Good point!